두정수사이의합.js

```diff
@@ -0,0 +1,15 @@
+function solution(a, b) {
+    let answer = 0;
+    
+    if (a <= b) {
+        for(let i = a; i <= b; i++) {
+            answer += i;
+        }
+    }
+    if (a > b) {
```

Comment:
```suggestion
    if (a <= b) {
        for(let i = a; i <= b; i++) {
            answer += i;
        }
    }
    else {
```
a <= b 의 여집합은 a > b 입니다  따라서 위 코드도 동작 하고
작성하신대로 if 문만 사용하면 위 조건에 만족해도 밑 조건까지 불필요하게 검사합니다